node-native/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "native"
```

Comment:
we were thinking about something like `lumina-node-uniffi` and `node-uniffi` directory. wdyt?
We already use `native` usually when we talk on just running lumina on host / desktop and I think it's a wording used in `lumina-cli`

Reply:
yeah i just saw that. Its definitely better, i'll rename it